### PR TITLE
improvement: added tooltip style prop to tooltop exit button

### PR DESCRIPTION
**Pull request by ionutrusu-bytex**
Added tooltip style props for lightbox component on the exit button

## Screenshots:
**Before** (If applicable):

**After**:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by ionutrusu-bytex** [comment]
@bfbiggs changed, thank you

File: react/src/lib/Lightbox/index.js [hunk 1]
```diff
@@ -581,6 +583,10 @@ Lightbox.defaultProps = {
   onChange: null,
   onClose: null,
   onDownload: null,
+  popoverProps:{
+    isContained:true,
+    direction: 'bottom-left'
```
Comment (holsted) [inline react/src/lib/Lightbox/index.js:588]: Why are we changing the default from bottom right to bottom left?

Comment (ionutrusu-bytex) [reply react/src/lib/Lightbox/index.js]: fixed @holsted

File: react/src/lib/Lightbox/index.js [hunk 2]
```diff
@@ -556,6 +556,8 @@ Lightbox.propTypes = {
   onDownload: PropTypes.func,
   /** @prop Array of Lightbox pages */
   pages: PropTypes.array.isRequired,
+  /** @prop tooltip style */
```
Comment (bfbiggs) [inline react/src/lib/Lightbox/index.js:559]: This line is just missing a pipe(|) with the default props like tooltips below. This is used for the docs site.

Comment (ionutrusu-bytex) [reply react/src/lib/Lightbox/index.js]: @bfbiggs fixed

**Comment by ionutrusu-bytex** [comment]
@holsted it passes on lightbox
image

**Pull request merged**